static/docs/commands-reference/pipeline_show.md

```diff
@@ -1,7 +1,8 @@
 # show
 
 Show [stages](/doc/commands-reference/run) in a pipeline that lead to the
-specified stage. By default it lists DVC-files (`.dvc` file extension).
+specified stage. By default it lists
+[DVC-files](/doc/user-guide/dvc-file-format).
```

Comment:
Let's create a ticket to automate this linking with JS.

Reply:
Created. I'm not sure about this though, please refer to https://github.com/iterative/dvc.org/issues/435#issuecomment-501410295